elasticsearch/src/main/java/org/hibernate/search/elasticsearch/settings/impl/model/AnalyzerDefinitionJsonAdapterFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.elasticsearch.settings.impl.model;
+
+import java.util.List;
+
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * @author Yoann Rodiere
+ */
+public class AnalyzerDefinitionJsonAdapterFactory extends AnalysisDefinitionJsonAdapterFactory {
```

Comment:
Actually, it occurred to me only *after* I spent hours on this crap :-(
Anyway... Using `JsonElement`/`JsonArray` may hurt when we'll implement validation for analyzer definitions, so I'd like to avoid that as much as possible. I know it was a real pain when I implemented mapping validation.